implementations/build-graalsqueak.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+if [ "$1" = "style" ]
+then
+  exit 0
+fi
+
+set -e # make script fail on first error
+readonly SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${SCRIPT_PATH}/script.inc"
+
+readonly GRAALSQUEAK_DIR="${SCRIPT_PATH}/GraalSqueak"
+readonly GRAALSQUEAK_JAR="https://github.com/hpi-swa/graalsqueak/releases/download/awfy/graalsqueak-component.jar"
```

Comment:
Looks like this is not versioned. Think it might make it easier to have some version part of the string to be able to track changes.

Reply:
Sure, that's one reason this PR is still WIP :)